catalog/core/catalog-core-api/src/main/java/org/codice/alliance/catalog/core/api/types/Isr.java

```diff
@@ -22,267 +22,267 @@
 public interface Isr {
 
     /**
-     * Attribute name for accessing the Cloud Cover percentage for this Metacard. <br/>
+     * <p> Attribute name for accessing the Cloud Cover percentage for this Metacard. </p>
```

Comment:
Instead of replacing all those `<br/>` with `<p></p>`, we could have removed the `<br/>` altogether. The JavaDoc tool already takes care of adding all the proper paragraph tags and such.